pkg/cmd/service/service.go

```diff
@@ -65,6 +65,9 @@ type Config struct {
 
 	// Logging. Possible options: text,json
 	LogFormat string `default:"text" split_words:"true"`
+
+	PlaygroundEnabled bool `default:"true" split_words:"true"`
```

Comment:
Do we want this on by default? Does it make sense to have it off by default, and add a command to the Makefile such as `run-with-playground` that turns it on.

Reply:
@jon-whit can you answer this one?

Reply:
I think we will need to update the docker-compose file as well. Expose port 3000 and, if we have this off by default, add an env var.

Reply:
Done